nimiSlides.nim

```diff
@@ -176,57 +182,71 @@ template initReveal*() =
     fragmentCore(endAnimations = @[@[endAnimation]]):
       body
 
-  template animateCode(lines: varargs[HSlice], body: untyped) =
+  template animateCode(lines: varargs[HSlice[int, int]], body: untyped) =
     ## Shows code and its output just like nbCode, but highlights different lines of the code in the order specified in `lines`.
     ## lines: Specify which lines to highlight and in which order. (Must be specified as a HSlice)
     ## Ex: 
     ## ```nim
     ## animateCode(1..1, 2..3, 5..5, 4..4): body
     ## ```
     ## This will first highlight line 1, then lines 2 and 3, then line 5 and last line 4.
-    discard
+    newNbBlock("animateCode", nb, nb.blk, body):
+      var linesString: string
+      if lines.len > 0:
+        linesString &= "|"
+      for line in lines:
+        linesString &= $line.a & "-" & $line.b & "|"
+      if lines.len > 0:
+        linesString = linesString[0 .. ^2]
+      nb.blk.context["highlightLines"] = linesString
+      captureStdout(nb.blk.output):
+        body
+
 
   template bigText(text: string) =
-    nbText: "<h2 class=\"r-fit-text\">" & text & "</h2>"
+    rawBlock: "<h2 class=\"r-fit-text\">" & text & "</h2>"
 
   template removeCodeOutput =
```

Comment:
I was thinking, well I could add this to nimib too (checking if "Code" is in blk.command). Then I realize that it should be equivalent to a `nb.blk.output = ""` and if we really want we could have that kind of template in nimib (`nbRemoveOutput`) just for ergonomics...

Reply:
Yes indeed, just haven't thought of any other case than code blocks where we want to remove the output. But I could add it as well to the `nbRaw*` PR if you'd like :)